update dates and ical file

mentalcaries opened the pull request:
For this PR I :
- updated the 2024 application window opening and closing dates
- updated the cohort kickoff date
- updated the regions
- updated the `.ical` file

segdeha commented on build/about-us/index.html:
```diff
@@ -440,6 +440,19 @@ <h2 id="mentors">Mentors</h2>
+                <li class="member-container">
+                    <img alt="Jill Kuchman" class="team-member-photo" src="/assets/img/members/jillkuchman.jpg">
+                    <div class="member-caption">
+                        <p>Jill/p>
```
Missing the `<` on the closing tag

segdeha commented on build/about-us/index.html:
```diff
@@ -440,6 +440,19 @@ <h2 id="mentors">Mentors</h2>
+                    </div>
+                </li>
+                <li class="member-container span-3">
+                    <p><strong>Jill Kuchman</strong> is a Quality Assurance Engineer at <a href="https://www.vacasa.com/">Vacasa</a> in Portland, OR. She's a tech education enthusiast who has written curriculum and taught at bootcamps in both Portland and Nairobi.</p>
```
Please swap out the `'` for an `’`

segdeha replied:
I’m super curious about your experience in Kenya!!!

segdeha commented on build/about-us/index.html:
```diff
@@ -440,6 +440,19 @@ <h2 id="mentors">Mentors</h2>
+                </li>
+                <li class="member-container span-3">
+                    <p><strong>Jill Kuchman</strong> is a Quality Assurance Engineer at <a href="https://www.vacasa.com/">Vacasa</a> in Portland, OR. She's a tech education enthusiast who has written curriculum and taught at bootcamps in both Portland and Nairobi.</p>
+                    <p>Jill enjoys creating in all forms and can usually be found knitting, drawing, out enjoying the sun, or trying out extravagant recipes.</p>
```
Love this!

jmkuchman commented:
Thanks, @segdeha! That should be all fixed up now.

mentalcaries committed:
update dates and ical file

mentalcaries committed:
point url to correct ical file

mentalcaries committed:
Update the Cohort region type

The linter throws an error for this locally for some reason, even though it's valid TS. Updating via GH's code editor to bypass it

mentalcaries committed:
point to correct iCal file

segdeha commented on public/ics/S1-2024-Application-Window.ics:
```diff
@@ -0,0 +1,39 @@
+UID:20221226T163918Z-828471998@marudot.com
+DTSTART;VALUE=DATE:20240121
+DTEND;VALUE=DATE:20240128
+SUMMARY:The Collab Lab Applications Open for Session 1 2024
```
Please change "Session 1" to "February". Thanks!

segdeha commented on public/ics/S1-2024-Application-Window.ics:
```diff
@@ -0,0 +1,39 @@
+DTEND;VALUE=DATE:20240128
+SUMMARY:The Collab Lab Applications Open for Session 1 2024
+URL:https://the-collab-lab.codes/participate/
+DESCRIPTION:Applications are now open for the Session 1, 2024 Collab Lab cohort. Go to our website to apply!\n\nLas postulaciones están abiertas para nuestro grupo de personas de S1 2024. Ve a nuestro sitio web para postularte!
```
Please change "Session 1" to "February" and "S1" to "febrero". Thanks!

segdeha commented on src/components/ApplicationPrompt.astro:
```diff
@@ -60,7 +60,7 @@ const regionList = listFormatter.format(cohortRegions);
 					💡 Want a reminder of our next application window?
 					<a
-						href="/ics/Q3-2023-Application-Window.ics"
+						href="/ics/S1-2024-Application-Window.ics"
```
Please change "S1" to "Feb". Thanks!

segdeha reviewed (approved):
Hey! Just a couple of wording changes for the iCal file. Otherwise, you're approved to merge once you make those changes. Thanks!

mentalcaries committed:
updating naming, session 2024 to february 2024

mentalcaries committed:
Update Cohort types
